bool kinect_on
---
bool was_set
